katharsis-core/src/main/java/io/katharsis/resource/registry/responseRepository/ResourceRepositoryAdapter.java

```diff
@@ -73,4 +95,20 @@ public JsonApiResponse delete(ID id, QueryParams queryParams) {
 	public Object getResourceRepository() {
 		return resourceRepository;
 	}
+
+	public FilterOperator getDefaultOperator() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)